test/unit/db/seeds_test.rb

```diff
@@ -90,14 +90,14 @@ def teardown
       assert master_service.metrics.find_by(system_name: system_name, unit: 'hit', friendly_name: description)
     end
 
-    assert(tenant_app_plan = master_service.application_plans.default)
+    assert (tenant_app_plan = master_service.application_plans.default)
```

Comment:
I prefer without the space between assert and `(` 😂 .. wdyt?

Reply:
https://github.com/3scale/porta/pull/1294#discussion_r330977927
OMG :joy:

Reply:
we should have a style guide hahaha